tests/python/relay/test_any.py

```diff
@@ -1084,5 +1142,41 @@ def test_any_adv_index():
     check_result([np_data, np_index, np_index], mod, ref_res)
 
 
+def verify_any_repeat(data_shape, np_dshape, repeats, axis):
+    mod = tvm.IRModule()
+    dtype = "float32"
+    data = relay.var("data", shape=data_shape, dtype=dtype)
+    y = relay.repeat(data, repeats, axis)
+    mod["main"] = relay.Function([data], y)
+    np_data = np.random.uniform(size=np_dshape).astype(dtype)
+    ref_res = np.repeat(np_data, repeats, axis)
+    check_result([np_data], mod, ref_res)
+
+
+def test_any_repeat():
```

Comment:
to let the CI pick up gpu test, we need to decorate the test_xx with `@tvm.testing.uses_gpu`